templates/packageVersion.mustache

```diff
@@ -66,6 +60,31 @@
           <p>None</p>
           {{/cabalFileRevisions}}
         </dd>
+        {{#pkgDesc.library}}
+        <dt class="col-sm-3">Dependencies</dt>
+        <dd class="col-sm-9">{{#build-depends}}{{#.}}{{.}}<br>{{/.}}{{/build-depends}}</dd>
+        {{/pkgDesc.library}}
+        </dd>
+        {{#pkgDesc.sub-libraries}}
+        <dt class="col-sm-3">sub-library {{name}}</dt>
```

Comment:
"sub-library" is a bit of a term of art: in the cabal file they're just "libraries"! I think it's probably clear enough to have "library <library name> dependencies" for all libraries, and then the library at the top with the same name as the package is clearly the main library?